src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/MlInformation.cs

```diff
@@ -0,0 +1,216 @@
+        /// <since_tizen> 12 </since_tizen>
+        public void SetNumber(string key, int value)
+        {
+            if (string.IsNullOrEmpty(key))
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "The property key is invalid");
+
+            NNStreamerError ret = NNStreamerError.InvalidParameter;
+
+            switch(_type)
+            {
+                case InfoType.Option:
+                    ret = Interop.Util.SetOptionValue(_handle, key, new IntPtr(value), null);
+                    break;
+                case InfoType.Information:
+                    ret = NNStreamerError.NotSupported;
+                    Log.Error(NNStreamer.TAG, "InfoType Iniformation does not support set value");
+                    break;
+            }
+
+            NNStreamer.CheckException(ret, "Failed to set string value");
+        }
+
+        private IntPtr GetInformationHandle(string key)
+        {
+            if (string.IsNullOrEmpty(key))
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "The property key is invalid");
+
+            IntPtr value = IntPtr.Zero;
+            NNStreamerError ret = NNStreamerError.InvalidParameter;
+
+            switch(_type)
+            {
+                case InfoType.Option:
+                    ret = Interop.Util.GetOptionValue(_handle, key, out value);
+                    break;
+                case InfoType.Information:
+                    ret = Interop.Util.GetInformationValue(_handle, key, out value);
+                    break;
+            }
+
+            NNStreamer.CheckException(ret, "Failed to get information handle from key");
+
+            return value;
+        }
+
+        /// <summary>
+        /// Gets a string value of key in mlInformation instance.
+        /// </summary>
+        /// <param name="key">The key to be set.</param>
+        /// <returns>The string value of the key.</returns>
+        /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public string GetString(string key)
+        {
+            IntPtr value = GetInformationHandle(key);
+            return Interop.Util.IntPtrToString(value);
+        }
+
+        /// <summary>
+        /// Gets a TensorsData value of key in mlInformation instance.
+        /// </summary>
+        /// <param name="key">The key to be set.</param>
+        /// <returns>The TensorsData value of the key.</returns>
+        /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public TensorsData GetTensorsData(string key, TensorsInfo info)
```

Comment:
Consider adding new API (internal or public?) to get tensors-info by using 'ml_tensors_data_get_info()' from data handle.
Then we can get the tensors-data instance without tensors-info instance.
```
public TensorsData GetTensorsData(string key)
```

Reply:
Oh, I really needed this API while adding TCs.
I think this makes easier to handle tensor-data.
I wil add this API and updates some TCs.
Thank you :)